src/Twig/Extension/MediaExtension.php

```diff
@@ -61,9 +61,9 @@ public function getTokenParsers()
     }
 
     /**
-     * @param MediaInterface $media
-     * @param string         $format
-     * @param array          $options
+     * @param MediaInterface|int|string $media
```

Comment:
this could receive the MediaInterface directly or an identifier, it is correct to type it with int|string?

Reply:
`MediaInterface|int|string` Seems correct to me